### DISCUSSION TITLE
使用R画OR(95%CI) 图

### DISCUSSION THREAD

**Discussion by JYL2303** [ShixiangWang/self-study#53] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/54757508/68730027-8ccc6f80-0606-11ea-99d7-c48c253ea11a.png)
类似这种，中间的点是OR值，或者RR值。我看文献上说是用R 3.3做的，但是我一直没办法找到具体的方法跑这种图，求指教

**Comment by ShixiangWang** [comment, marked as answer]
你这个问题的重点在哪里？是不知道生成这个数据吗？还是有这种数据但不会画这个图？还是需要画坐标轴截断的图呢？

- 如果你的重点是第一个，应该寻找计算 OR 和 RR 值的方法并应用到自己的数据上
- 如果第第二个的话，我推荐你使用 ggpubr 包，这个简单，请看 https://rpkgs.datanovia.com/ggpubr/reference/ggerrorplot.html
- 如果是第三种问题，可以阅读 http://www.dataguru.cn/article-4827-1.html 和 https://www.jianshu.com/p/0e4fa8849479

有问题进一步讨论。

祝好，
诗翔

**Comment by JYL2303** [comment]
重点是第二个问题。感谢大神解答~~
